### PR TITLE
Update bower_components path to support polytool serve

### PR DESCRIPTION
**Pull request by abdonrd**
Update `bower_components` path to support `polytool serve`.

The new `generator-polymer-init` already use these paths: https://github.com/PolymerLabs/generator-polymer-init/blob/master/generators/app/templates/application/index.html

### CODE REVIEW

**Comment by abdonrd** [comment]
@justinfagnani it is correct, right? :)

File: index.html [hunk 1]
```diff
@@ -52,7 +52,7 @@
         if (!webComponentsSupported) {
           var script = document.createElement('script');
           script.async = true;
-          script.src = '../webcomponentsjs/webcomponents-lite.min.js';
+          script.src = 'bower_components/webcomponentsjs/webcomponents-lite.min.js';
```
Comment (justinfagnani) [inline index.html]: This should be an absolute URL if index.html is loaded from any other path, as in shell apps that serve index from `/foo/bar`.

Comment (justinfagnani) [inline index.html]: The HTML import for psk-app should be absolute too

**Comment by justinfagnani** [comment]
LGTM

**Comment by abdonrd** [comment]
Update with Justin comments!

**Comment by addyosmani** [comment]
👍

**Pull request merged**